lighthouse-core/test/lib/page-functions-test.js

```diff
@@ -226,7 +226,8 @@ describe('Page Functions', () => {
 
     it('returns node path through shadow root', () => {
       const el = document.createElement('div');
-      const main = el.appendChild(document.createElement('main'));
+      const main = document.createElement('main');
```

Comment:
main -> mainEl (any place we don't do that for element variables is a mistake, like L233)